src/web/app/components/comment-box/comment-table/comment-table.component.ts

```diff
@@ -70,9 +72,28 @@ export class CommentTableComponent implements OnInit {
   @Output()
   updateCommentEvent: EventEmitter<number> = new EventEmitter();
 
-  constructor() { }
+  constructor(private tableComparatorService: TableComparatorService) { }
 
   ngOnInit(): void {
+    this.sortComments();
+  }
+
+  ngOnChanges(): void {
+    this.sortComments();
+  }
+
+  /**
+   * Sorts the comments by last edited date.
+   */
+  sortComments(): void {
```

Comment:
The sorting should be done at the root page? (single point of truth?)

Reply:
There's quite a bit of pages that uses the comment table component other than session result ones and those pages don't have comments pre-processed. So i was thinking if there was an alternative way to do the sorting? Or would the only option be to preprocess comments at these pages too?

Reply:
The main concert here is we break the single point of truth. The parent component losses control on the model.

> There's quite a bit of pages

There are only two if I am not wrong? We don't need the help page example box to be sorted.

Reply:
Got it. then will stick with preprocessing the comments since help page doesn't need to be sorted